scripts/add-cluster-role-to-team.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+if [ $# -ne 3 ] ||
+        [ "$1" == "--help" ] ||
+        [ "$1" == "-h" ] ||
+        ([ $1 != "admin" ] && [ $1 != "edit" ] && [ $1 != "view" ]); then
+    echo "Binds a given role to a group for specified namespace"
+    echo -e "\nUsage: $0 admin/edit/view NAMESPACE OWNER-TEAM\n"
+    exit 0
+fi
+
+APP_NAME="cluster-scope"
```

Comment:
I think that thanks to @larsks you can simplify this now :slightly_smiling_face: 

You can use `scripts/common.sh` to get access to shared variables:
https://github.com/operate-first/apps/pull/633/files#diff-edcce2a3d4c8083832ab865dbe7b27bc52d1d2c91cb23e86ff5bb8aa25d0c13bR8-R10